lib/wallaby/element.ex

```diff
@@ -151,8 +153,9 @@ defmodule Wallaby.Element do
     case driver.set_value(element, value) do
       {:ok, _} ->
         element
-      {:error, :stale_reference_error} ->
+      {:error, :stale_reference} ->
         raise Wallaby.StaleReferenceException
+      e -> e
```

Comment:
I know I'm pedantic, but I'd like thos to say... element or error, whichever it is :D